lib/kafka/consumer.rb

```diff
@@ -203,7 +203,7 @@ def each_message(min_bytes: 1, max_bytes: 10485760, max_wait_time: 1, automatica
         max_wait_time: max_wait_time,
       )
 
-      consumer_loop do
+      consumer_loop(automatically_mark_as_processed: automatically_mark_as_processed) do
```

Comment:
It seems like we could make `automatically_mark_as_processed` an instance variable and avoid passing it to `consumer_loop`, but I'm not sure.